src/execution_plan/ops/op.c

```
@@ -44,27 +54,27 @@ void OpBase_Init
 	op->childCount     = 0;
-	op->op_initialized = false;
 
 	// function pointers
 	op->init     = init;
 	op->free     = free;
 	op->clone    = clone;
 	op->reset    = (reset) ? reset : _OpBase_reset_noop;
-	op->profile  = NULL;
-	op->consume  = consume;
+	op->consume  = init ? _OpBase_consume : consume;
+	op->op_consume = consume;
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)